controller/src/beerocks/master/db/db.h

```diff
@@ -155,10 +156,11 @@ class db {
     bool has_node(sMacAddr mac);
 
     bool add_virtual_node(sMacAddr mac, sMacAddr real_node_mac);
-    bool add_node(std::string mac, std::string parent_mac = std::string(),
-                  beerocks::eType type         = beerocks::TYPE_CLIENT,
-                  std::string radio_identifier = std::string());
-    bool remove_node(std::string mac);
+    bool add_node(const sMacAddr &mac,
```

Comment:
According to [CppCoreGuideline F.16](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-in), a small object should be past by value instead of const-ref, and they say that 'small' is two-three words. Since sMacAddr is 6 bytes, it is less than two words, so I'd say it's small. And indeed, I can imagine that passing it by reference creates overhead due to additional pointer dereferences.

Reply:
I accidentally pushed and merged before committing those changes, so I've created #331 instead.